README.md

```diff
@@ -77,6 +77,10 @@ If using the coreMQTT port for the MQTT, the following needs to be included and
 
 Other than these, your choice of libraries for TLS and TCP/IP are up to you to choose and properly configure/compile.
 
+### Using AWS FreeRTOS
+
+We have a guide [here](./docs/aws_freertos_port.md) which will help you port and build your AWS FreeRTOS project with the Azure IoT middleware for FreeRTOS.
```

Comment:
Can we say : Porting your application based on amazon-freertos to connect to AzureIoT services using middleware